politeiawww/api/pi/v1/v1.go

```diff
@@ -159,6 +160,18 @@ const (
 	ErrorStatusVoteParamsInvalid
 	ErrorStatusBallotInvalid
 	ErrorStatusVotePageSizeExceeded
+
+	// Vote cast errors
```

Comment:
```suggestion
	// Cast vote errors
```

Since the struct is a `CastVote`